src/components/ErrorBanner/ErrorBanner.less

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// ErrorBanner Styles
+
+
+@ErrorBanner-height: 52px;
+@ErrorBanner-iconSize: 40px;
+
+.ms-ErrorBanner {
+  position: relative;
+  border-bottom: 1px solid @ms-color-neutralSecondaryAlt;
+  background-color: @ms-color-themeLighterAlt;
+  width: 100%;
+  height: @ErrorBanner-height;
+  .ms-font-s();
```

Comment:
Sometimes we like to put mixins like this at the top (above position: relative) to make it a little cleaner and easier to parse but this is probably just a preference thing.